### DISCUSSION TITLE
[Feature Request]: Remove "Mails" from the list of services

### DISCUSSION THREAD

**Discussion by peterbrinck** [beyondcode/herd-community#351] [Ideas]
### Feature Description

When not using Herd Pro, the "Mails" item is still shown in the list of services
It also pushes "FPM-X.X" to the bottom, so you have to scroll in order to see it

I think it would make sense to remove "Mails" if the user doesn't have Herd Pro or move it to the bottom

![CleanShot 2024-02-06 at 10 52 08](https://github.com/beyondcode/herd-community/assets/6154068/5140ecc2-4790-4057-b388-1a083f38b66b)

### Is this feature valuable for other users as well and why?

All users who doesn't have Herd Pro

**Comment by sschlein** [comment]
That makes a lot of sense, I think that slipped through during UX testing!
